eng/targets/DiaSymReaderNative.targets

```diff
@@ -0,0 +1,35 @@
+<!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
+<Project>
+
+  <PropertyGroup>
+    <!-- 
+      Ensure the binaries using DiaSymReader.Native are marked as AnyCPU unless they specify 
+      a different target. This should be the default behavior but recent SDK changes to more 
+      correctly consider RIDs caused our behavior to change here. Once the SDK logic is settled
+      here we can remove this.
+
+      https://github.com/dotnet/sdk/issues/3495
+    -->
+    <PlatformTarget Condition="'$(PlatformTarget)' == ''">AnyCPU</PlatformTarget>
```

Comment:
❔ Have we verified that this occurs before whatever operation was causing the SDK to set this property to x86?

Reply:
Yes.